packages/atomic/package.json

```diff
@@ -34,7 +34,7 @@
     "cypress:test": "./node_modules/cypress/bin/cypress run --record --key 0e9d8bcc-a33a-4562-8604-c04e7bed0c7e --browser chrome",
     "cypress:test:firefox": "./node_modules/cypress/bin/cypress --record --key 0e9d8bcc-a33a-4562-8604-c04e7bed0c7e --browser firefox",
     "cypress:open": "./node_modules/cypress/bin/cypress open",
-    "npm:publish": "npm publish --access public"
+    "npm:publish": "node ../../scripts/deploy/is-published.js . || npm publish --access public"
```

Comment:
I would add a `--no-verify` as pointed out in slack, at least just for confirm if that's the problem or not. For atomic/headless/bueno.

If that's not causing the issue, then we could remove it.